test/Microsoft.Azure.SignalR.Tests/MultiEndpointServiceConnectionContainerTests.cs

```diff
@@ -1347,7 +1350,8 @@ public async Task TestEndpointManagerWithMultiHubsWithServerEndpointUpdate()
             await Task.WhenAll(containers.Select(x => x.MockReceivedServersPing("aaa;bbb")));
             containers1 = container1.GetTestOnlineContainers();
             await Task.WhenAll(containers1.Select(x => x.MockReceivedServersPing("aaa;bbb")));
-            await Task.Delay(6000);
+            await hubEndpoints.Single(x => x.Name == "1" && x.ServerEndpoint.AbsoluteUri == testSe).ScaleTask;
+            await Task.Delay(100);
```

Comment:
Is there still need to delay?

Reply:
Yes. Little delay required as there's a few operations after the `ScaleTask` completes which is to sync the endpoints changes in the other side. So minor delay is preferred to ensure the validation success.